### PR TITLE
UI & a11y updates for the title bar icon buttons

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
https://github.com/user-attachments/assets/a9dd0d66-fb3b-46a0-b889-5f31ddbe2067

### CODE REVIEW

File: src/components/iconbutton.js [hunk 1]
```diff
@@ -25,21 +26,35 @@ export class IconLinkButton extends LitElement {
     href: { attribute: true },
     alt: { attribute: true },
     icon: { attribute: true },
+    onClicked: { attribute: true },
```
Comment (strseb) [inline src/components/iconbutton.js:29]: Add it to the docstring :)

File: src/ui/browserAction/popupPage.js [hunk 2]
```diff
@@ -270,12 +276,13 @@ export class BrowserActionPopup extends LitElement {
         : null}
     `;
   }
-  static backBtn(back) {
-    return html`<img
+  static backBtn() {
+    return html` <mz-iconlink
```
Comment (strseb) [inline src/ui/browserAction/popupPage.js:280]: No need for events to bubble up to window.
```
window.addEventListener("goBack", () => {back();});
```
Consider: 
```js
 static backBtn(back) {
    return html` <mz-iconlink @goBack=${back} [...]>  </mz-iconlink>`
```

File: src/components/iconbutton.js [hunk 3]
```diff
@@ -25,21 +26,35 @@ export class IconLinkButton extends LitElement {
...
-    browser.tabs.create({
-      url: this.href,
-    });
+    switch (this.onClicked) {
+      case "open-link":
+        browser.tabs.create({
+          url: this.href,
+        });
+        break;
+      case "go-back":
```
Comment (strseb) [inline src/components/iconbutton.js:45]: Wondering if we need a custom event vs just setting href to null, (checking that in onclick) and just have the parent component have an  `@click=${goBack()}` listener.

**Review by strseb — APPROVED** [review]
r+wc

File: src/components/iconbutton.js (hunk 1 above)
Comment (lesleyjanenorton) [reply src/components/iconbutton.js]: Removed :)

**Pull request merged**